cms/djangoapps/contentstore/tests/test_course_listing.py

```diff
@@ -93,11 +95,11 @@ def test_errored_course_global_staff(self):
         """
         GlobalStaff().add_users(self.user)
 
-        course_key = self.store.make_course_key('Org1', 'Course1', 'Run1')
+        course_key = CourseLocator('Org1', 'Course1', 'Run1')
         self._create_course_with_access_groups(course_key, self.user)
 
-        with patch('xmodule.modulestore.mongo.base.MongoKeyValueStore', Mock(side_effect=Exception)):
-            self.assertIsInstance(modulestore().get_course(course_key), ErrorDescriptor)
+        with patch('xmodule.modulestore.split_mongo.split_mongo_kvs.SplitMongoKVS', Mock(side_effect=Exception)):
+            self.assertIsInstance(self.store.get_course(course_key), ErrorDescriptor)
```

Comment:
Can you `ddt` this test, rather than just changing it? I don't really want to lessen our coverage of old mongo.